stylesheets/livingstyleguide/_color-swatches.scss

```diff
@@ -26,12 +26,38 @@
     z-index: -1;
   }
 
+  &:hover:before {
+    color: black;
+    cursor: pointer;
+  }
+
   &:after {
+    cursor: pointer;
     display: block;
     font-weight: normal;
     text-align: center;
   }
 }
+.-lsg-copy-color{
+  background-color: hsla(333, 100%, 0%, 0.6);
+  border-radius:7%;
+  color: #FFF;
+  cursor: pointer;
+  font-family: 'Georgia', 'Times New Roman', serif;
+  font-size: 0.8em;
+  height:2em;
```

Comment:
Remove the height please, it crops the text for me:

<img width="169" alt="bildschirmfoto 2015-07-19 um 18 17 06" src="https://cloud.githubusercontent.com/assets/103399/8766682/60296d70-2e42-11e5-897f-dca63a4fccb1.png">